From b5def7a58660914b49c1d9269ca55a5efac1cafb Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Tue, 28 Feb 2023 17:09:37 +0100
Subject: [PATCH 412/529] ARM: configs: remove useless configuration on
 stm32mp1x

Update fragment-01-multiv7_cleanup.config to remove:
CONFIG_CXD2880_SPI_DRV
CONFIG_MTD_NAND_PL35X
CONFIG_PWM_ATMEL_TCB

Signed-off-by: Eric Fourmont <eric.fourmont-ext@st.com>
Change-Id: Ifb4d40f4d267168d6b8b5e6f6b4d5527b09ba069
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/291980
---
 arch/arm/configs/fragment-01-multiv7_cleanup.config | 3 +++
 1 file changed, 3 insertions(+)

--- a/arch/arm/configs/fragment-01-multiv7_cleanup.config
+++ b/arch/arm/configs/fragment-01-multiv7_cleanup.config
@@ -46,6 +46,7 @@
 # CONFIG_CAN_MCP251X is not set
 # CONFIG_CAN_RCAR is not set
 # CONFIG_CHROME_PLATFORMS is not set
+# CONFIG_CXD2880_SPI_DRV is not set
 # CONFIG_DETECT_HUNG_TASK is not set
 # CONFIG_DVB_A8293 is not set
 # CONFIG_DVB_AF9013 is not set
@@ -223,6 +224,7 @@
 # CONFIG_MEDIA_TUNER_XC4000 is not set
 # CONFIG_MEDIA_TUNER_XC5000 is not set
 # CONFIG_MICROCHIP_PHY is not set
+# CONFIG_MTD_NAND_PL35X is not set
 # CONFIG_MTD_SPI_NOR_USE_4K_SECTORS is not set
 # CONFIG_PCI is not set
 # CONFIG_PCI_DOMAINS_GENERIC is not set
@@ -232,6 +234,7 @@
 # CONFIG_POWER_AVS is not set
 # CONFIG_POWER_SUPPLY is not set
 # CONFIG_PREEMPT_NONE is not set
+# CONFIG_PWM_ATMEL_TCB is not set
 # CONFIG_REGULATOR_WM8994 is not set
 # CONFIG_SCHED_DEBUG is not set
 # CONFIG_SECTION_MISMATCH_WARN_ONLY is not set
